### ISSUE TITLE
**Pakistan - Any plan for adding Public holidays for Pakistan**

### ISSUE THREAD

**Issue by Yasir-Khan22** [nager/Nager.Date#580]
### More information

I really want to see additional support for Pakistan's public holidays. 
If yes. Tell me If I should raise a PR for this.?

**Comment by tinohager** [comment]
Hi,

https://en.wikipedia.org/wiki/Public_holidays_in_Pakistan

- The state holidays look possible
- For religious holidays, as in other countries, it will be difficult because the moon position would have to be calculated here

**Comment by Yasir-Khan22** [comment]
Yes, obviously for religious holidays it will totally depend on moon sighting.
Tell me should I raise PR for state holidays only now?
